code/datums/autolathe/guns.dm

```diff
@@ -131,14 +131,18 @@
 	build_path = /obj/item/weapon/gun/projectile/automatic/z8
 
 /datum/design/autolathe/gun/wintermute
-	name = "FS BR .20 \"Wintermute\""
+	name = "FS AR .20 \"Wintermute\""
 	build_path = /obj/item/weapon/gun/projectile/automatic/wintermute
 
+/datum/design/autolathe/gun/kovacs
```

Comment:
I moved this from the .30 section for consistency. I forgot to add the "SA BR .20" shit.